src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -205,32 +205,50 @@ def _parse_emissions(emissions: Dict[str, EmissionResult]) -> Dict[str, libecalc
         }
 
     @staticmethod
-    def extract_requested_pressures_from_model(
-        energy_usage_model: Dict[datetime, EnergyUsageModel],
+    def extract_input_compressor_pressures_from_models(
```

Comment:
```suggestion
    def get_requested_compressor_pressures(
```

I dont like the naming, easily confused with "inlet_pressure" imo. Changing it to the name above makes it more agnostic of "how it is done", and more focused on "what is it that we get"...what do you think?

Reply:
I agree